force_wfmanager/ui/review/i_base_plot.py

```diff
@@ -0,0 +1,20 @@
+from chaco.api import ArrayDataSource
+from traits.api import Instance
+
+from .i_data_view import IDataView
+
+
+class IBasePlot(IDataView):
+    """Envisage required interface for the BasePlot class.
```

Comment:
These comments are a bit misleading - it isn't required by Envisage.  Interfaces like this are more of an opportunity for things that don't inherit from `BasePlot` to still participate/be contributed/etc. as-if they do.

So more neutral wording like "Interface definition for plots."

Also, should this be `IPlot`, not `IBasePlot`?

Reply:
> These comments are a bit misleading - it isn't required by Envisage. Interfaces like this are more of an opportunity for things that don't inherit from `BasePlot` to still participate/be contributed/etc. as-if they do.
> 
> So more neutral wording like "Interface definition for plots."

Agreed. The docstrings of all the `Interface` classes introduced have been amended as suggested. They also point to the force-bdss documentation for instructions as to how to implement them.

> Also, should this be `IPlot`, not `IBasePlot`?

The `IBasePlot` subclass was originally intended as an interface around the generic 2D scatter plot defined by `BasePlot` - the `Plot` class just adds an extra option for colouring data points by another variable. However, this naming seems a little confusing, so I've now renamed the `Plot` class to `ColorPlot` and included it in a separate file `color_plot.py`.

Reply:
Update: `Plot` --> `ColorPlot` refactoring has been lifted to a new branch: `enh/color-plot`